staking/programs/staking/src/error.rs

```diff
@@ -59,6 +59,8 @@ pub enum ErrorCode {
     PositionSerDe,
     #[msg("Position out of bounds")] //6027
     PositionOutOfBounds,
-    #[msg("Other")] //6028
+    #[msg("Can't vote during an account's transfer epoch")] //6028
+    VoteDuringTransferEpoch,
```

Comment:
(i guess you don't care about backward compatibility since this isn't deployed yet?)

Reply:
yes, this is accurate